core/ui/compose/designsystem/src/main/kotlin/app/k9mail/core/ui/compose/designsystem/atom/RadioGroup.kt

```diff
@@ -0,0 +1,40 @@
+package app.k9mail.core.ui.compose.designsystem.atom
+
+import androidx.compose.foundation.layout.Row
+import androidx.compose.material3.RadioButton
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import app.k9mail.core.ui.compose.designsystem.atom.text.TextLabelLarge
+import kotlinx.collections.immutable.ImmutableList
+
+@Composable
+fun <T> RadioGroup(
```

Comment:
There should be a preview to investigate the RadioGroup in the design system. 

Also the component should be added to the Catalog app in the atom section.